ServerCore/Pages/Teams/Index.cshtml

```diff
@@ -8,7 +8,7 @@
 <h2>All Teams</h2>
 
 <p>
-    @if (Model.Event.IsTeamRegistrationActive) // Or current user is admin
+    @if (Model.Event.IsTeamRegistrationActive && Model.EventRole == ModelBases.EventRole.admin)
```

Comment:
There was a change since the comment - this page is only going to be for authors & admins, so if you're not an admin you can't create a team (since authors can't create teams)